python/utilities/log_list/openssl_generator.py

```diff
@@ -0,0 +1,25 @@
+def _enabled_logs_conf(logs):
+    return "enabled_logs = %s\n" % (", ".join(log["url"] for log in logs))
+
+def _log_conf(log, operators):
+    return (
+        "[%(name)s]\n"
+        "description = %(description)s\n"
+        "key = %(key)s\n"
+        "url = https://%(url)s\n"
+        "operated_by = %(operated_by)s\n" % {
+            "name": log["url"],
```

Comment:
Yes, the URL is guaranteed to be unique and is perhaps more readable than the log description when many are concatenated together for the "enabled_logs" line.